test_conformance/math_brute_force/macro_unary_float.cpp

```diff
@@ -456,7 +454,8 @@ static cl_int Test(cl_uint job_id, cl_uint thread_id, void *data)
             }
 
 
-            for (k = MAX(1, gMinVectorSizeIndex); k < gMaxVectorSizeIndex; k++)
```

Comment:
Looks like this is variable shadowing the `k` in the outer loop, don't think it makes a difference for correctness but naming the two loop induction variables differently might be more readable

Reply:
Nice observation! I have to admit I had to resort to some automation for this patch and didn't look at this aspect. I'm actually tempted to keep the same name for now because there's a bigger issue with the two loops: there should be only one, most likely. I think other files might be affected too. I'll make a note to investigate this further.

Reply:
sounds good, thanks